tests/commands/test_cli.py

```diff
@@ -0,0 +1,25 @@
+import unittest
+
+from click.testing import CliRunner
+from tutor.commands.cli import *
```

Comment:
Please avoid start imports. Instead, import the `cli` module and refer to all objects as attributes of this module. Same comment for all test files below.

Reply:
@regisb, if I understand you correctly you meant the star/asterisk `*` import on line 4? I already resolved that for now if that's what you meant.